sycl/doc/extensions/Matrix/dpcpp-joint-matrix.asciidoc

```diff
@@ -101,13 +126,14 @@ enum class matrix_layout {
 }
 ```
 
-Intel AMX and DPAS hardware require B matrix to be in VNNI or 32 bits packed layout. If we multiply matrices A (M, K) and B (K, N) into a matrix C (M, N). The logical sizes are M, K, N. However, the packed shape for B tile uses the VNNI format, which is described below. The user must provide the information of packed_b layout to make the implementation allocate the right shape. The layout information for Intel AMX should be specified in user code as follows: 
+Intel AMX and DPAS hardware require B matrix to be in VNNI or 32 bits packed layout.  However, the packed shape for B tile uses the VNNI format, which is described below. The user must provide the information of packed_b layout to make the implementation allocate the right shape. The layout information for Intel AMX should be specified in user code as follows:
 
 ```c++
-joint_matrix<int8_t, K, N, packed_b> tB(sg);
+joint_matrix<int8_t, matrix_use::b, K, N, packed_b> tB(sg);
```

Comment:
To be consistent:
reverse to the original definition,
add a subsection how this would be written for tensorcores (add use argument)